internal/indexer/db/client/params.go

```diff
@@ -53,6 +53,7 @@ func (db *IndexerDatabase) GetBbnStakingParams(ctx context.Context) ([]*indexert
 			DelegationCreationBaseGasFee: stakingParams.DelegationCreationBaseGasFee,
```

Comment:
above there is `MaxActiveFinalityProviders` property that was added back in 2024, we don't have this field on indexer side, but I left it as is - maybe it's still relevant